cloudbuild.yaml

```diff
@@ -2,33 +2,94 @@
 ## AWS Sources ##
 #################
 steps:
-## Build DynamoDB source
+## Pull images to speed up build
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |
+    docker pull gcr.io/$PROJECT_ID/awsdynamodb:latest || exit 0
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: 'bash'
+  args:
+  - '-c'
+  - |
+    docker pull gcr.io/$PROJECT_ID/awssqs:latest || exit 0
```

Comment:
We can parallelize the pull with `waitFor: ['-']` which indicates that this step begins immediately.
ref: https://cloud.google.com/cloud-build/docs/configuring-builds/configure-build-step-order